db/database.sql

```diff
@@ -25,18 +25,19 @@ CREATE TABLE `user` (
 	PRIMARY KEY (`id`),
 	UNIQUE KEY `username` (`username`),
 	UNIQUE KEY `email` (`email`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
 
 DROP TABLE IF EXISTS `user_group`;
 CREATE TABLE `user_group` (
 	`id` int(10) unsigned NOT NULL AUTO_INCREMENT,
 	`user_id` int(10) unsigned NOT NULL,
 	`group_id` int(10) unsigned NOT NULL,
 	`manager` tinyint(1) NOT NULL DEFAULT '0',
+        `mailbox` tinyint(1) NOT NULL DEFAULT '0',
```

Comment:
Changes to database schemas need to also have an update file, like the other .sql files in this directory, otherwise the changes won't be applied to existing installations. The `config` table's `value` attribute should be updated to match that name, both in this file and at the end of the update script.

Reply:
Ok, I will do that